sentry.properties

```diff
@@ -1,4 +1,4 @@
 defaults.url=https://sentry.io/
 defaults.org=plant-for-the-planet
 defaults.project=planet-donations
-# cli.executable=../path/to/bin/sentry-cli
\ No newline at end of file
+cli.executable=../../../.npm/_npx/a8388072043b4cbc/node_modules/@sentry/cli/bin/sentry-cli
```

Comment:
@norbertschuler are you sure the path will be same in all node environments?

Reply:
No. I am not. Does it run for you?

Reply:
We can omit this change for this patch. I am not using the sentry cli anyway yet.